### DISCUSSION TITLE
Source Azure Blob Storage: Unstructured file type error

### DISCUSSION THREAD

**Discussion by vvead** [airbytehq/airbyte#34006] [Connector Questions]
Hello!

Reporting here as we're trying to setup the Azure Blob source connector to extract [unstructured file type data](https://airbyte.com/blog/airbyte-now-supports-extracting-text-from-documents) (pdfs and docx mainly)

✅ Fetching PDF/DOCX/PPTX contents from Gdrive using the provided connector works (source-gdrive v0.0.3)
✅ Fetching CSV contents from Azure Blob using the provided connector (source-azureblobstorage v0.2.3) works
❌ Fetching PDF/DOCX/PPTX contents from Azure Blob using the provided connector (source-azureblobstorage v0.2.3) fails

We're new to Airbyte, sorry if we overlooked some obvious things

Relevant log output
```
Unable to connect to stream <airbyte_cdk.sources.file_based.stream.default_file_based_stream.DefaultFileBasedStream object at 0x7f519ea143d0> - Traceback (most recent call last):
 File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/file_based/file_based_source.py", line 80, in check_connection
 ) = stream.availability_strategy.check_availability_and_parsability(stream, logger, self)
 File "/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/file_based/availability_strategy/default_file_based_availability_strategy.py", line 64, in check_availability_and_parsability
 handle.close()
```

Thanks!

**Comment by marcosmarxm** [comment]
Thanks for reporting this @vvead are you reading the same files from g-drive and azure blob storage?

[a comment by an automated account is not shown]
